asserts/model.go

```diff
@@ -657,6 +694,113 @@ func checkOptionalSystemUserAuthority(headers map[string]interface{}, brandID st
 	return nil, fmt.Errorf("%q header must be '*' or a list of account ids", name)
 }
 
+func checkModelValidationSetAccountID(headers map[string]interface{}, what, brandID string) (string, error) {
+	accountID, err := checkOptionalStringWhat(headers, "account-id", what)
+	if err != nil {
+		return "", err
+	}
+
+	// default to brand ID if account ID is not provided
+	if accountID == "" {
+		return brandID, nil
+	}
+	return accountID, nil
+}
+
+// checkModelValidationSetSequence reads the optional 'sequence' member, if
+// not set, returns 0 as this means unpinned. Unfortunately we are not able
+// to reuse `checkSequence` as it operates inside different parameters.
+func checkModelValidationSetSequence(headers map[string]interface{}, what string) (int, error) {
+	seq, err := checkIntWithDefaultWhat(headers, "sequence", what, 0)
+	if err != nil {
+		return 0, err
+	}
+
+	// Assert that sequence number is not zero, as zero will be interpreted as not specified,
+	// which means that the validation-set is unpinned.
```

Comment:
the comment is confusing, as we can return 0

Reply:
Good point!